scripts/india_lgd/local_government_directory_districts/preprocess.py

```diff
@@ -119,9 +119,9 @@ def _load_and_format_lgd(self):
         # if there are NA values then replace it with '' character
         self.lgd_df = pd.read_csv(self.lgd_csv, dtype=str)
         self.lgd_df.fillna('', inplace=True)
-        # Drop title rows in the top and empty rows after 740.
-        # The actual data is between 2nd and 740th row. So keep only them.
-        self.lgd_df = self.lgd_df.iloc[1:739]
+        # Drop title rows in the top and empty rows after 741.
+        # The actual data is between 2nd and 741st row. So keep only them.
+        self.lgd_df = self.lgd_df.iloc[1:741]
```

Comment:
is there a way to future proof this (so we won't have to keep updating these constants, which someone else might forget to do)..

- perhaps you could call read_csv with header=None or header=2
- i'm also surprised there are empty rows - the csv looks fairly clean and doesn't include empty lines (or you could use skip_blank_lines in read_csv).

Reply:
@beets  Thank you. Makes sense. Done.